### ISSUE TITLE
Exception when a union of two polygons return a multipolygon

### ISSUE THREAD

**Issue by aalih** [hzi-braunschweig/SORMAS-Project#2286]
### Bug Description
To simplify the geo shapes, SORMAS union polygons that have a common border, but the union of two polygons can return a Multipolygon and SORMAS expect a Polygon. This throws a ClassCastExecption stopping the server.

### Expected Behavior
If the union of two polygons returns a multipolygon the server should capture the exception and continue working

[multipolygon-error-diff.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/4753143/multipolygon-error-diff.txt)

**Comment by MateStrysewskeSym** [comment]
@aalih Can you tell me how exactly this can be reproduced?

**Comment by aalih** [comment]
Hi @MateStrysewskeSym, I had the problem while incorporating the Cuba shapefiles, I can send them to you to reproduce the error, just tell me how.

BTW, i think it will be better if the shapefiles can be provided without recompiling the code, using the sormas.properties file to point to the shapefiles directory and document the expected layout and file names. I can open a new issue with this suggestion if you want.

**Comment by MateStrysewskeSym** [comment]
@aalih Yes, that would be great! However, instead of specifying them in the sormas.properties file, I'd probably prefer to have a shape file import directly in the system because that would be a lot more user-friendly. We actually have this on our radar, but haven't had the opportunity to implement it yet.

**Comment by aalih** [comment]
Hi @MateStrysewskeSym you can find the cuba shapefiles here: https://github.com/aalih/SORMAS-Project/tree/infomed_fixes/sormas-backend/src/main/resources/shapefiles/cuba

Your solution to add the shapefiles to the system is definitely better than mine ;)
